fieldservice_recurring_repair/models/fsm_recurring.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2024 Camptocamp SA (https://www.camptocamp.com).
+# @author: Italo Lopes <italo.lopes@camptocamp.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class FSMRecurringOrder(models.Model):
+    _inherit = "fsm.recurring"
+
+    def _split_create_by_equipment(self, vals):
+        """
+        Create an order for each equipment.
+        On fieldservice_repair, we create a repair order for each equipment.
+        """
+        self.ensure_one()
+        orders = self.env["fsm.order"]
+        for equipment in self.equipment_ids:
+            order_vals = vals.copy()
+            order_vals["equipment_id"] = equipment.id
+            orders |= self.env["fsm.order"].create(order_vals)
+        return orders
```

Comment:
```suggestion
        return self.env["fsm.order"].create(
            [
                dict(vals, equipment_id=equipment.id)
                for equipment in self.equipment_ids
            ]
        )
```

nitpicking: to create all at once (better for perf)